pems/districts/views.py

```diff
@@ -1,15 +1,24 @@
 from django.views.generic import TemplateView
 
+from .models import District
+
 
 class IndexView(TemplateView):
     template_name = "districts/index.html"
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["district_queryset"] = District.objects.all()
+        return context
+
 
 class DistrictView(TemplateView):
     template_name = "districts/district.html"
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         district = self.kwargs.get("district")
-        context["district"] = district
+        context["url_district"] = district
```

Comment:
This seems like the... `district.number`? Why not just call it that?

https://github.com/compilerla/pems/pull/94#pullrequestreview-2777126322